docs/design/2021-03-09-dynamic-privileges.md

```diff
@@ -233,14 +233,16 @@ No change
 | `SYSTEM_VARIABLES_ADMIN` | Allows changing any GLOBAL system variable. | Currently this required `SUPER`. It will now require `SYSTEM_VARIABLES_ADMIN` or `SUPER`. |
 | `ROLE_ADMIN` | Allows granting and revoking roles. | Won’t allow revoking on restricted_users (see below). |
 | `CONNECTION_ADMIN` | Allows killing connections. | Like `PROCESS` static privilege, but slightly more restrictive (no show processlist). |
+| `SYSTEM_USER` | The user can't be altered or dropped by only the `CREATE USER` privilege. | Helps prevent security escalations. |
```

Comment:
Any link to this change? I have no idea of that.

Reply:
That's right. It's not merged, but adding this one makes a lot of sense. See: https://mysqlserverteam.com/the-system_user-dynamic-privilege/